src/workloads/docs/CrudActorEncrypted.yml

```diff
@@ -0,0 +1,212 @@
+SchemaVersion: 2018-07-01
+Owner: "@10gen/server-security"
+Description: |
```

Comment:
Since this is a core genny feature change, not just a workload-specific one, I'd recommend putting some docs for this in the [using](https://github.com/mongodb/genny/blob/master/docs/using.md) docs, and linking to this example workload.

Reply:
Good point. I will do that as a follow-up ticket.